elements/bulbs-pdfobject/bulbs-pdfobject.js

```diff
@@ -21,7 +21,7 @@ class BulbsPDFObject extends BulbsHTMLElement {
   embedPDF () {
     let poster = this.getAttribute('poster');
     let options = {
-      fallbackLink: `<a href='[url]'><img src="${poster}"></a>`,
+      fallbackLink: `<a href='[url]'><div class="fallback-btn">Tap to explore document</div><img src="${poster}"></a>`,
```

Comment:
provides a hook for scss setup on the integrating app. could be generic but doesn't hurt to provide a more semantic class to hang things on